### ISSUE TITLE
No keyboard access to settings options

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3973]
**Description**:
Most of the different settings options cannot be accessed with the keyboard. The settings options only appear when the menu button has focus or is active. Once a keyboard user moves away from the button, the settings options disappear. This makes it impossible for a keyboard user to navigate from a settings menu option into the set of controls for that option.

For example, when tabbing to the "Beta features" menu item in the settings modal, it’s not possible to get to the Beta toggle button using the keyboard. As soon as tab focus moves to the next menu item, "Commands," the Beta features options disappear and are replaced with the Commands options.

**WCAG Criteria:**
[2.1.1 Keyboard - A](https://www.w3.org/WAI/WCAG21/Understanding/keyboard)

Screenshot or Screen Recording:
[Screen Recording](https://share.zight.com/8Lue7Jwz)

Notes:
This is currently set up as a tab panel, but the UI does not fully support keyboard users. Following the [ARIA Tab Panel pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/) to provide access to keyboard users where:

Arrow keys navigate through the tab list.
The tab key moves the user through the active tab item and corresponding tab panel controls.
